src/convertToObject.js

```diff
@@ -6,7 +6,19 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const stylesList = sourceString.replaceAll('\n', ' ').split(';');
+  const stylesObject = {};
+  const filtredStylesList = stylesList.filter((val) => val.trim().length > 0);
+
+  for (let i = 0; i < filtredStylesList.length; i++) {
+    filtredStylesList[i] = filtredStylesList[i].split(':');
+    filtredStylesList[i][0] = filtredStylesList[i][0].trim();
+    filtredStylesList[i][1] = filtredStylesList[i][1].trim();
```

Comment:
This line assumes that `filtredStylesList[i][1]` exists. If a style does not have a value (e.g., 'color:'), this will result in an error. Consider checking if `filtredStylesList[i][1]` is undefined before attempting to trim it.